client/templates/shared/course-card.html

```diff
@@ -7,7 +7,8 @@
                 </a>
             </div>
             <div class="caption">
-                <a class="course-category" href="#">Mathematics</a>
+              <a class="course-category" href="#">Mathematics</a> 
+              <a class="course-tags" href="/learn/{{keywords.[0]}}">{{keywords.[0]}}</a>  
```

Comment:
This should be reverted to `course-category`, since course tags are displayed in the course card footer.